config/consensus.go

```diff
@@ -25,9 +25,12 @@ import (
 	"github.com/algorand/go-algorand/protocol"
 )
 
-// ConsensusParams specifies settings that might vary based on the
+// ConsensusParams is a reference to a ConsensusParamsVal.
+type ConsensusParams *ConsensusParamsVal
```

Comment:
I think you want a type alias here since these are supposed to be identical types:
```suggestion
type ConsensusParams = *ConsensusParamsVal
```

Here is some reference information: https://go.dev/talks/2016/refactor.article#TOC_5.1.